Makefile.dep

```diff
@@ -297,7 +304,6 @@ endif
 
 ifneq (,$(filter nhdp,$(USEMODULE)))
```

Comment:
Dito. This is even propably just a merge error resulting from the (temporary) merge in this PR